app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_arguments = command.split(" ")
```

Comment:
There is no need to indicate that you should split on space, it is a default option. Leave just `command.split()`.